config/regex.js

```diff
@@ -22,6 +22,8 @@ module.exports = {
     STEP_NAME: /^[\w-]+$/,
     // Jobs can only be named with A-Z,a-z,0-9,-,_
     JOB_NAME: /^[\w-]+$/,
+    // PR JOB Name can only be PR-1 or PR-1:main, gourp1: PR-prNum, group2: prNum, gourp3: jobName
+    PR_JOB_NAME: /^(PR-(\d+))(?::([\w-]+))?$/,
```

Comment:
Is job name `meow-blah` still legal?
I don't think this regex captures that case